onnx_chainer/testing/test_onnxruntime.py

```diff
@@ -61,12 +61,17 @@ def check_output(model, x, filename, out_key='prob', opset_version=None):
             ' chainer.Variable itself. But a {} object was given.'.format(
                 type(x)))
 
+    rt_out_key = None
     if isinstance(chainer_out, (list, tuple)):
         chainer_out = (y.array for y in chainer_out)
     elif isinstance(chainer_out, dict):
-        chainer_out = chainer_out[out_key]
-        if isinstance(chainer_out, chainer.Variable):
-            chainer_out = (chainer_out.array,)
+        if isinstance(out_key, str):
```

Comment:
[optional] I don't think `check_output`, which is only for testing, needs to be flexible. I think it's better to keep `out_key` has a single type. How about changing `out_key` to `out_keys` and changing its default value to `['prob']`?

Reply:
fix, (along with mxnet test script)

Reply:
I think default value of `out_key` could be confused for some case, and this value is used in only mxnet_test script before. This functions if for test, so I'll change default value to `None`.